src/session-summaries/what-is-cors.md

```diff
@@ -0,0 +1,20 @@
+----
+hosts: Innocent Amadi, Valentino Ugbala
+guests: 
+summarized_by: Chimdia Anyiam
+summarized_by_social_media: https://twitter.com/AnyiamChimdia
+----
+# What is Cors
+
+
+
+The hosts take us through what CORS (Cross Origin Resource Sharing) is and how it comes to play when developing web applications. CORS is a mechanism that uses additional HTTP headers to tell a browser to let a web application running at one origin have permission to access selected resources from a server at a different origin.
```

Comment:
Update the paragraph to this instead. Feel free to add to it or change some things if you think it's necessary.

_In this video, Valentino explains Cross-Origin Resource Sharing (CORS) and its implications when developing web applications. CORS is a way to let a web application, running in the browser on one origin, have permission to access selected resources from a server at a different origin. By default, browsers only allow access to resources from the same origin (Same Origin Policy). CORS is implemented using HTTP headers._

Reply:
Thanks, you actually made that paragraph sound much better and I have learnt a few things.